paasta_tools/cli/cmds/status.py

```diff
@@ -838,13 +843,23 @@ def print_flink_status(
         f" {status.overview['slots-available']}/{status.overview['slots-total']} slots available"
     )
 
+    # Avoid cutting job name. As opposed to default hardcoded value of 32, we will use max length of job name
+    max_job_name_length = max([len(get_flink_job_name(job)) for job in status.jobs])
+    # Limiting longest allowed job name to be 120 chars. This is to ensure while printing on cli the formatting is maintained on normal screens
+    allowed_max_job_name_length = min(
```

Comment:
Please see the last paragraph of the previous comment about this code.
`allowed_max_job_name_length` is width of the first column only, it cannot be equal to the width of the terminal, but should be less of it, otherwise  there will be no space for `State`, `Job ID`, and `Started` columns.
`allowed_max_job_name_length` need to be reduced so there will be space for these columns.

Reply:
Please check the calculations once again. Sorry for misunderstanding.